lib/order.rb

```diff
@@ -0,0 +1,54 @@
+require_relative 'texting_twilio'
+
+class Order
+
+  attr_reader :basket, :texting_service
+
+  def initialize(texting_service = TextingTwilio)
+    @basket = {}
+    @texting_service = texting_service
+  end
+
+  def add_to_basket(menu, dish_number, quantity)
+    dish = menu.select(dish_number)
+    @basket[dish] ||= 0
+    quantity.times do
+      @basket[dish] += dish.price
+    end
+  end
+
+  def summary
+    fail "Basket empty" if empty_basket?
+    summary_string = ""
+    basket.each_pair do |dish, cost|
+      summary_string << "#{(cost / dish.price).round}x #{dish.name} | £#{format('%.2f', cost)}\n"
```

Comment:
Metrics/LineLength: Line is too long. [97/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)